### ISSUE TITLE
chiflights22 is not a CRAN package

### ISSUE THREAD

**Issue by JosiahParry** [rstudio/bslib#1046]
This app is not reproducible without being able to sleuth around the internet very well. It seems chiflights is a passion project from @cpsievert (perhaps made specifically for this example!) and is not published on CRAN. Figuring out how to  install this package was a bit of work that some people less familiar with R might not know how to do. 

https://github.com/rstudio/bslib/blob/5046bbd4aadfc758903290fed0584bb8711ef13a/inst/examples/flights/app.R#L6

**Comment by gadenbuie** [comment]
That's right, it's made with the [anyflights package](https://github.com/simonpcouch/anyflights), which expressly recommends _not sending_ a derivative package to CRAN.

We should definitely include at least a note in a comment in the source, I've also run into a 🤔 moment trying to remember where the package lives.

For the record, it's recorded here in bslib's DESCRIPTION file

https://github.com/rstudio/bslib/blob/5046bbd4aadfc758903290fed0584bb8711ef13a/DESCRIPTION#L64

**Comment by JosiahParry** [comment]
now _that_ is what i call a description file! holy smokes!
